### PR TITLE
Moved IPAs of words with tildes to multiple lines

### PR DESCRIPTION
**Pull request by sonofthomp**
Fixed [issue #379](https://github.com/CUNY-CL/wikipron/issues/379).

Replaced lines with multiple tilde-separated pronunciations of the same word on one line with each pronunciation on a separate line.

- [X] Updated `Unreleased` in `CHANGELOG.md` to reflect the changes in code or data.

### CODE REVIEW

**Comment by kylebgorman** [comment]
This makes sense to me though we should probably work it into the lower levels eventually. 

The tests are failing because, I think, you need to rerun `./postprocess`?

**Comment by sonofthomp** [comment]
Fixed the issues with the tests.

The problem was that I had accidentally put [the summary.tsv file](https://github.com/CUNY-CL/wikipron/blob/master/data/scrape/summary.tsv) in the [`tsv/` directory](https://github.com/CUNY-CL/wikipron/tree/master/data/scrape/tsv), so it was being parsed as a file of phonemes.

**Review by kylebgorman — APPROVED** [review]
LGTM minus two nits. Nice to see the `./postprocess` is working well for you here.

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Unreleased
 
 #### Added
 
-=======
+-   Caught iso639.language.LanguageNotFoundError error in codes.py (\#498)
```
[the remark this answers is not in the document]
Comment (sonofthomp) [reply CHANGELOG.md]: Sure, added!

**Pull request merged**